src/test/java/uk/nhs/hee/tis/trainee/credentials/api/PlacementResourceTest.java

```diff
@@ -0,0 +1,153 @@
+    cred1.setIssuedAt(Instant.parse("2017-02-03T10:37:30Z"));
+    cred1.setRevokedAt(Instant.parse("2018-02-03T10:37:30Z"));
+
+    cred2.setCredentialId("e2a26a50-2c51-4327-8692-1cb05g32af22");
+    cred2.setTraineeId("47165");
+    cred2.setTisId("2bd3f009-9c7a-4352-bcc6-9dff8eac07u9");
+    cred2.setCredentialType("TRAINING_PLACEMENT");
+    cred2.setIssuedAt(Instant.parse("2011-02-03T10:37:30Z"));
+    cred2.setRevokedAt(null);
+  }
+
+  @Test
+  void testSuccessfullyCallTheService() throws Exception {
+
+    String tisId = "47165";
+    String token = TestJwtUtil.generateTokenForTisId(tisId);
+
+    when(service.getTheLatestCredential(CredentialType.TRAINING_PLACEMENT, tisId))
+        .thenReturn(List.of(cred1));
+
+    mockMvc.perform(get("/api/placement")
+            .header(HttpHeaders.AUTHORIZATION, token)
+            .contentType(MediaType.APPLICATION_JSON))
+        .andExpect(status().isOk())
+        .andExpect(content().contentType(MediaType.APPLICATION_JSON))
+        .andExpect(jsonPath("$.[*].credentialId").value("e2a26a50-2c51-4327-8692-1cb05b80af07"))
+        .andExpect(jsonPath("$.[*].traineeId").value("47165"))
+        .andExpect(jsonPath("$.[*].tisId").value("2bd3f009-9c7a-4352-bcc6-9dff8eac07d6"))
+        .andExpect(jsonPath("$.[*].credentialType").value("TRAINING_PLACEMENT"))
+        .andExpect(jsonPath("$.[*].issuedAt").value("2017-02-03T10:37:30Z"))
+        .andExpect(jsonPath("$.[*].revokedAt").value("2018-02-03T10:37:30Z"));
+
+  }
+
+  @Test
+  void testReturnListOfResultsWithCorrectValues() throws Exception {
+    String tisId = "47165";
+    String token = TestJwtUtil.generateTokenForTisId(tisId);
+
+    when(service.getTheLatestCredential(CredentialType.TRAINING_PLACEMENT, tisId))
+        .thenReturn(List.of(cred1, cred2));
+
+    mockMvc.perform(get("/api/placement")
+            .header(HttpHeaders.AUTHORIZATION, token)
+            .contentType(MediaType.APPLICATION_JSON))
+        .andExpect(status().isOk())
+        .andExpect(content().contentType(MediaType.APPLICATION_JSON))
+        .andExpect(jsonPath("$.[0].credentialId").value("e2a26a50-2c51-4327-8692-1cb05b80af07"))
+        .andExpect(jsonPath("$.[0].traineeId").value("47165"))
+        .andExpect(jsonPath("$.[0].tisId").value("2bd3f009-9c7a-4352-bcc6-9dff8eac07d6"))
+        .andExpect(jsonPath("$.[0].credentialType").value("TRAINING_PLACEMENT"))
+        .andExpect(jsonPath("$.[0].issuedAt").value("2017-02-03T10:37:30Z"))
+        .andExpect(jsonPath("$.[0].revokedAt").value("2018-02-03T10:37:30Z"))
+        .andExpect(jsonPath("$.[1].credentialId").value("e2a26a50-2c51-4327-8692-1cb05g32af22"))
+        .andExpect(jsonPath("$.[1].traineeId").value("47165"))
+        .andExpect(jsonPath("$.[1].tisId").value("2bd3f009-9c7a-4352-bcc6-9dff8eac07u9"))
+        .andExpect(jsonPath("$.[1].credentialType").value("TRAINING_PLACEMENT"))
+        .andExpect(jsonPath("$.[1].issuedAt").value("2011-02-03T10:37:30Z"))
```

Comment:
Extract these values in to constants so they can be reused, as well as making sure we don't make a typo in the expectations and break the whole test, the naming makes the expectations easier to read.

e.g.
```java
.andExpect(jsonPath("$.[0].credentialId").value(CREDENTIAL_ID_1))
...
.andExpect(jsonPath("$.[1].credentialId").value(CREDENTIAL_ID_2))
```

Is much clearer than what we currently have.